simulator/authorization_manager.go

```diff
@@ -215,15 +232,26 @@ func (m *AuthorizationManager) RemoveAuthorizationRole(req *types.RemoveAuthoriz
 	return body
 }
 
-func updateSystemPrivileges(privileges []string) []string {
-OUT:
-	for _, spr := range systemPrivileges {
-		for _, pr := range privileges {
-			if spr == pr {
-				continue OUT
-			}
+func (m *AuthorizationManager) privIDs(ids []string) ([]string, *soap.Fault) {
+	system := make(map[string]struct{}, len(m.system))
+
+	for _, id := range ids {
+		if _, ok := m.privileges[id]; !ok {
+			return nil, Fault("", &types.InvalidArgument{InvalidProperty: "privIds"})
```

Comment:
ok.. so privileges in this list is reserved? (cannot be used?)

Reply:
See: http://pubs.vmware.com/vsphere-6-5/index.jsp#com.vmware.wssdk.apiref.doc/vim.AuthorizationManager.html

"They are statically defined and never change for a single version of a product"